paper.tex

```diff
@@ -73,9 +73,9 @@ \section*{Introduction}
 on analysing the mathematical properties of these trees [cite?], and numerous
 methods exist to infer them~\citep{felsenstein2004inferring}. The situation is
 quite different in recombining organisms, where the analysis of genetic data
-depends on summaries ultimately derived from the trees such as allele
+depends on summaries ultimately derived from the trees~\citep{ralph2020efficiently}
 % TODO cite the other papers that preceded ralph2020efficiently here.
-frequencies~\citep{ralph2020efficiently}, site frequency spectra, patterns of linkage
```

Comment:
This positioning of the citation was intentional: all of these things are references to papers that go through the equivalance of that particular thing with trees. Putting this first makes it look like ralph2020efficiently explains things like PCA etc, which it doesn't.

Reply:
Ah, I see. I moved it because I feel like we need a citation that to back up the "summaries ultimately derived from the trees" quote, and the "duality" paper applies to the SFS too. But I'll put it back and we can discuss. Is there a paper out there that makes the point about the summaries of whatever sort being derived from trees?

Reply:
> Is there a paper out there that makes the point about the summaries of whatever sort being derived from trees?

Not that I know of off the top of my head, but it'll be referred to in the stats paper if it does. If you're looking, it would be helpful to get the other papers talking about allele frequencies and trees from the references, like the comment mentions.

Reply:
Perhaps we just merge the freq + SFS items:

```
where the analysis of genetic data
depends on summaries ultimately derived from the trees
such as allele frequencies and site frequency spectra ~\citep{ralph2020efficiently}, patterns of linkage
disequilibrium~\citep{mcvean2002genealogical} and principal
components~\citep{mcvean2009genealogical}
```

Reply:
> Not that I know of off the top of my head, but it'll be referred to in the stats paper if it does. If you're looking, it would be helpful to get the other papers talking about allele frequencies and trees from the references, like the comment mentions.

The relevant section of the stats paper says this:
> This duality (we will use the term more precisely later) has been used extensively in deriving properties of statistics in models of random mating (e.g., Tajima 1983; Tavaré 1984; Fu 1995), as has the fact that it applies regardless of the underlying demographic model (e.g., Gillespie and Langley 1979; Hudson 1983; Slatkin 1991; McVean 2002; Lohse et al. 2016; Ralph 2019)

Reply:
These are the ones I was thinking of:

> Suppose we want to calculate one of the linear functions of the frequency spectrum discussed by Fu (1995) and Achaz (2009),

Reply:
There's also [Felsenstein 1992](https://www.cambridge.org/core/journals/genetics-research/article/estimating-effective-population-size-from-samples-of-sequences-inefficiency-of-pairwise-and-segregating-sites-as-compared-to-phylogenetic-estimates/C199D5519B61D02C7052EAB262FA7449)

Reply:
The (Tajima 1983; Tavaré 1984; Fu 1995) cites are the right ones here. E.g. from Tavaré:

> The purpose of this paper is to review some of the pertinent mathematical details about genealogical processes, lines of descent, and their structure and demonstrate how such ideas can be applied to produce a wide variety of classical results in the mathematical theory of population genetics.